src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -468,6 +481,75 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         driver.swipe(from.getX(), from.getY(), to.getX(), to.getY(), SWIPE_DURATION_MILLIS);
         return this;
     }
+  
+    private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection direction, float percentage, int duration) {
+        Dimension screen = driver.manage().window().getSize();
+
+        Point from;
+        if(corner != null) {
+            int x;
+            int y;
+            switch(corner) {
+                case TOP_LEFT:
+                    x = screen.getWidth() - 100;
+                    y = screen.getHeight() - 10;
+                    from = new Point(x,y);
+                    break;
+                case TOP_RIGHT:
+                    x = screen.getWidth() - 10;
+                    y = screen.getHeight() - 10;
+                    from = new Point(x,y);
+                    break;
+                case BOTTOM_LEFT:
+                    x = screen.getWidth() - 100;
```

Comment:
the left side of the screen can be very different from `width - 100`. A value like `10` would probably be close what you want. In general though I think we should do this with percentage based margins from the border.
I believe @fuzzybinary mentioned he has a approach like this in the works. Would be great to get his opinion on this.